src/main/groovy/de/felixschulze/gradle/HockeyAppUploadTask.groovy

```diff
@@ -147,11 +153,13 @@ class HockeyAppUploadTask extends DefaultTask {
             }
         }
 
-        uploadAppplicationFileToHockeyApp(applicationFile, mappingFile, appId)
+        applicationFiles.each {
+            uploadApplicationFileToHockeyApp(it, mappingFile, appId)
```

Comment:
Hi @x2on any update on the release? We are really looking forward to upload multiple APKs on HockeyApp.